src/UI/Component/Listing/DescriptiveList.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+
+namespace ILIAS\UI\Component\Listing;
+
+/**
+ * Interface DescriptiveList
+ * @package ILIAS\UI\Component\Listing
+ */
+interface DescriptiveList extends \ILIAS\UI\Component\Component {
+
+    /**
+     * @param string[] $items
```

Comment:
When first reading this I wondered where the names of the items for the descriptive list would come from. Maybe you could clarify this here like:

@param string[] $items    string => string

or something like that? This would apply to the according factory method as well.